web/codegen.ts

```diff
@@ -25,6 +25,7 @@ const config: CodegenConfig = {
           URL: "string",
           Lang: "string",
           TranslatedString: "{ [lang in string]?: string } | null",
+          JSON: "any",
```

Comment:
Why is this change necessary?

Reply:
We've added a new Scalar named `JSON` in the backend for couple of things. Its better to put it here.